aiohttp_jinja2/__init__.py

```diff
@@ -81,3 +87,19 @@ def wrapped(*args):
             return response
         return wrapped
     return wrapper
+
+
+@asyncio.coroutine
+def context_processors_middleware(app, handler):
+    def middleware(request):
+        request[REQUEST_CONTEXT_KEY] = {}
+        for processor in app[APP_CONTEXT_PROCESSORS_KEY]:
+            request[REQUEST_CONTEXT_KEY].update(
+                (yield from processor(request)))
```

Comment:
If you do care about overlapping keys in `render_string` you should care here also.